sections/Dataset.tex

```diff
@@ -0,0 +1,61 @@
+\section{Dataset}\label{sec:dataset}
+Nordjyske is a Danish news agency that maintains multiple newspapers, radios, and other news sources throughout north Jutland, a region in Denmark.
+They store their news articles in a non-public database, where each article contains multiple metadata fields which describe some aspect of the data e.g. the author.
+The dataset we use ranges from 2017 to 2019 and contains $248,385$ articles.
+
+We perform some basic preprocessing to make the data more applicable for topic models.
+Firstly, because the dataset includes articles from multiple cities and regions, duplicates do occur in the dataset.
+These duplicates are removed, so only unique articles are kept.
+After this, we filter out words that appear in less than 10 articles and words that appear in more than 10$\%$ of articles.
+This is done to keep words that are used enough to find patterns in topics and to remove words that are similar to stop words.
+Finally, after the words are filtered out, the empty documents are removed.
+After preprocessing, the dataset contains $139,060$ articles that use a vocabulary of $69,192$ unique words.
+
+The metadata fields that we will be working with do have some problems that can be mitigated to a degree by preprocessing.
+These problems are all related to some metadata values having only a few documents.
+Since the metadata values will be used to group documents together and find common topics and words within grouped documents, metadata values that group too few documents are not very relevant and will therefore be removed.
```

Comment:
```suggestion
Since the metadata values will be used to group documents together and find common topics and words within grouped documents, metadata values that group too few documents are not very relevant and will therefore be combined or removed.
```